CefSharp.Core/ManagedCefBrowserAdapter.cpp

```diff
@@ -0,0 +1,32 @@
+#include "Stdafx.h"
+#include "Cef.h"
+
+double ManagedCefBrowserAdapter::GetZoomLevel()
+{
+    auto browser = _clientAdapter->GetCefBrowser();
+
+    if (browser != nullptr)
+    {
+        auto host = browser->GetHost();
+        if (CefCurrentlyOn(TID_UI))
+        {
+            return host->GetZoomLevel();
+        }
+        else
+        {
+            // TODO: Add an async version of GetZoomLevel at some point.
+            // NOTE: Use of ManualResetEvent is required here in order
+            // for simple marshaling of some kind of synchronization primitive
+            // to the callback.
+            Task<double>^ task = Cef::UIThreadTaskFactory->StartNew(gcnew Func<double>(this, &ManagedCefBrowserAdapter::GetZoomLevel));
```

Comment:
> WPF doesn't actually call GetZoomLevel atm. Seems a bit odd.

I didn't work until your previous PR, so there wasn't much point calling it. It just kept an internal reference to the amount that you incremented it by. It could now be updated.